reana-cwl.yaml

```diff
@@ -0,0 +1,7 @@
+version: 0.2.0
```

Comment:
0.6.0 since we are trying with latest REANA version, here and in the other YAML files